File: Android_App/JavaApp/DsmMealReviewApp/app/src/main/java/com/example/dsmmealreviewapp/MainActivity.java
package com.example.dsmmealreviewapp;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.ImageButton;

public class MainActivity extends AppCompatActivity {

    private ImageButton restaurent;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        restaurent = (ImageButton) findViewById(R.id.restaurent);

        restaurent.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(MainActivity.this, CheckingReview.class));
            }
        });
    }


}

File: Android_App/KotlinApp/ClickerGame/app/src/main/java/com/example/clickergame/MainActivity.kt
package com.example.clickergame

import android.icu.number.IntegerWidth
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.TextView
import android.widget.Toast
import com.example.clickergame.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {

    private lateinit var binding: ActivityMainBinding

    private var count = 0

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

        binding.plus.setOnClickListener {
            count++
            binding.tvResult.setText("" + count)
        }

        binding.minus.setOnClickListener {
            if (count == 0) {
                Toast.makeText(
                    this@MainActivity,
                    "주의! : 저희 게임에서는 0이하로\n          내려갈 수 없습니다.",
                    Toast.LENGTH_SHORT
                ).show()
            } else {
                count--
                binding.tvResult.setText("" + count)
            }
        }

        binding.reset.setOnClickListener {
            count = 0
            binding.tvResult.setText("" + count)
        }
    }
}